### PR TITLE
[OBMIN-120] Changed CSS to Styled-components 

### PR DESCRIPTION
**Pull request by smilaya**
Changed CSS to Styled-components ( Components - Nav_Main)

### CODE REVIEW

File: src/main/resources/react/src/components/header/nav_main/index.jsx [hunk 1]
```diff
@@ -1,49 +1,49 @@
...
 import { getLang } from 'store/auth/slice';
+import { route } from 'routes/routeConstants';
 import { ButtonAdv } from 'components/common/buttons';
 import { getTranslatedText } from 'components/local/localization';
 import { ReactComponent as SearchSvg } from 'assets/icons/search.svg';
 
 import NavCategory from '../nav_category';
 
-import './navMain.scss';
+// import './navMain.scss';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/header/nav_main/index.jsx:11]: remove pls

File: src/main/resources/react/src/components/header/nav_main/styles.js [hunk 2]
```diff
@@ -0,0 +1,169 @@
+import styled from 'styled-components';
+import logo from 'assets/img/logo.png';
+import { Link } from 'react-router-dom';
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/header/nav_main/styles.js:3]: sort by length
for example
import { Link } from 'react-router-dom';
import styled from 'styled-components';

import logo from 'assets/img/logo.png';

File: src/main/resources/react/src/components/header/nav_main/styles.js [hunk 3]
```diff
@@ -0,0 +1,169 @@
...
+`;
+
+export const InputSearch = styled.input`
+  max-width: 541px;
+  width: 100%;
+  height: 50px;
+  outline: none;
+
+  border: 1px solid ${({ theme: { colors } }) => colors.btnBlueActive};
+  border-radius: 25px;
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/header/nav_main/styles.js:146]: ${({ theme }) => theme.colors.btnBlueActive};

File: src/main/resources/react/src/components/header/nav_main/styles.js [hunk 4]
```diff
@@ -0,0 +1,169 @@
...
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  top: 5px;
+  left: 5px;
+  box-sizing: border-box;
+  border-radius: 20px;
+  width: 40px;
+  height: 40px;
+  background-color: ${({ theme: { colors } }) => colors.btnBlueActive};
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/header/nav_main/styles.js:167]: same

File: src/main/resources/react/src/components/header/nav_main/styles.js [hunk 5]
```diff
@@ -0,0 +1,169 @@
...
+export const WrapCategoriesBottom = styled.div`
+  position: relative;
+
+  &:after {
+    position: absolute;
+    right: -18px;
+    bottom: 4px;
+    content: '';
+    border: 5px solid transparent;
+    border-top: 5px solid ${({ theme: { colors } }) => colors.blackColorText};
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/header/nav_main/styles.js:107]: ${({ theme }) => theme.colors.btnBlueActive};

File: src/main/resources/react/src/components/header/nav_main/styles.js [hunk 6]
```diff
@@ -0,0 +1,169 @@
...
+`;
+
+export const WrapCategories = styled.div`
+  position: absolute;
+  height: 56px;
+  font-family: Roboto, sans-serif;
+  font-size: 16px;
+  font-weight: bold;
+  line-height: 24px;
+  color: ${({ theme: { colors } }) => colors.blackColorText};
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/header/nav_main/styles.js:53]: ${({ theme }) => theme.colors.btnBlueActive};

File: src/main/resources/react/src/components/header/nav_main/styles.js [hunk 7]
```diff
@@ -0,0 +1,169 @@
...
+
+  &:after {
+    position: absolute;
+    right: -18px;
+    bottom: 9px;
+    content: '';
+    border: 5px solid transparent;
+    border-bottom: 5px solid ${({ theme: { colors } }) => colors.btnBlueActive};
+  }
```
Comment (Wolshebnik) [inline src/main/resources/react/src/components/header/nav_main/styles.js:91]: if in component exist 2 callbacks and more
we need 

${({ theme }) => css`
color: ${({ theme }) => theme.colors.blackColorText};

 &:after {
    position: absolute;
    right: -18px;
    bottom: 9px;
    content: '';
    border: 5px solid transparent;
    border-bottom: 5px solid ${({ theme }) => theme.colors.btnBlueActive};
  }
`}

**Review by Wolshebnik — APPROVED** [review]

**Review by cloudmenson — APPROVED** [review]

File: src/main/resources/react/src/components/header/nav_main/styles.js (hunk 4 above)
Comment (Wolshebnik) [reply src/main/resources/react/src/components/header/nav_main/styles.js]: ${({theme}) => theme.colors.btnBlueActive}

**Review by Wolshebnik — APPROVED** [review]

**Pull request merged**